niceml/dlframeworks/tensorflow/callbacks/csvlogger.py

```diff
@@ -26,13 +26,37 @@ class CSVLogger(Callback):
     def __init__(
         self, experiment_context: ExperimentContext, filename: str = "train_logs.csv"
     ):
+        """Callback that streams epoch results to a CSV file.
```

Comment:
I suggest to not just repeat the class docstring. Write 'Initialize CSVLogger with a given Context and csv filename' or something similar instead.

Reply:
Done